pombola/core/static/js/hover-dropdown.js

```diff
@@ -0,0 +1,69 @@
+$(function(){
+
+    /*
```

Comment:
Alternatively, we could remove the comment entirely, and put the documentation into `_hover-dropdown.scss`